### PR TITLE
Update SPA JS and add orgs example

### PR DESCRIPTION
**Pull request by adamjmcgrath**
### Description

- Update SPA JS
- Make sure SPA JS organization docs appear in auth0-react docs
- Add an organizations example

### References

image

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -260,3 +261,42 @@ export const Profile = () => {
   );
 };
 ```
+
+## 5. Accept a user invite and login to an organization
```
Comment (frederikprijck) [inline EXAMPLES.md:265]: Should we have guidance around logging in to an org without accepting a user invitation? I think both SPAJS and Angular SDK treat these separately from one another.

Comment (adamjmcgrath) [reply EXAMPLES.md:265]: I have guidance for that on line 294, I'd like to keep all the guidance in 1 example if I can. Also, encouraging people to do `loginWithRedirect({ organization: 'foo' });` is going to lead to user's with broken silent auth

Comment (adamjmcgrath) [reply EXAMPLES.md]: Updated 4b90ad6

**Review by frederikprijck — APPROVED** [review]

**Comment by jackrr** [comment]
Thanks for this! Would love to leverage the new redirect method param introduced in https://github.com/auth0/auth0-spa-js/pull/717. When does this get published to npm?

**Comment by jackrr** [comment]
Looks like it went out with https://github.com/auth0/auth0-react/pull/223

**Pull request merged**